Source/WebCore/rendering/svg/RenderSVGResource.cpp

```diff
@@ -54,28 +55,24 @@ static inline bool inheritColorFromParentStyleIfNeeded(RenderElement& object, bo
 
 static inline RenderSVGResource* requestPaintingResource(RenderSVGResourceMode mode, RenderElement& renderer, const RenderStyle& style, Color& fallbackColor)
 {
-    const SVGRenderStyle& svgStyle = style.svgStyle();
-
-    bool isRenderingMask = renderer.view().frameView().paintBehavior().contains(PaintBehavior::RenderingSVGMask);
-
-    // If we have no fill/stroke, return nullptr.
-    if (mode == RenderSVGResourceMode::ApplyToFill) {
-        // When rendering the mask for a RenderSVGResourceClipper, always use the initial fill paint server, and ignore stroke.
-        if (isRenderingMask) {
-            RenderSVGResourceSolidColor* colorResource = RenderSVGResource::sharedSolidPaintingResource();
-            colorResource->setColor(SVGRenderStyle::initialFillPaintColor().absoluteColor());
-            return colorResource;
-        }
+    bool applyToFill = mode == RenderSVGResourceMode::ApplyToFill;
 
-        if (!svgStyle.hasFill())
-            return nullptr;
-    } else {
-        if (!svgStyle.hasStroke() || isRenderingMask)
+    // When rendering the mask for a RenderSVGResourceClipper, always use the initial fill paint server, and ignore stroke.
```

Comment:
This comment is duplicated inside the if-statement. Please rephrase it, or delete the comments inside the if-statement.

Reply:
On it. :-)

Reply:
Rephrasing it to remove 'ignore stroke' part since it is repeating samething as you pointed out.

Reply:
Done, pushed new commit. Marking this as "Resolve" and will add "merge-queue".